.github/workflows/createReleaseBranch.yml

```diff
@@ -45,9 +45,9 @@ jobs:
         with:
           email: ${{ secrets.IDEE_GH_EMAIL }}
       - name: Set NPM at the correct version for Lerna
-        run: npm install -g npm@8.12.1
+        run: npm install -g npm@@9.8.1
       - run: npm ci
-      - run: npm install -g shelljs && npm install -g lerna
+      - run: npm install -g shelljs && npm install -g lerna@5.5.4
```

Comment:
Can see identical changes and passing GHA run in my fork 
https://github.com/forcedotcom/salesforcedx-vscode/commit/da5f94338e62fd0ced8652fe26384aa8ccfdfc68

https://github.com/gbockus-sf/salesforcedx-vscode/actions/runs/6643056792